starknet_py/tests/e2e/docs/quickstart/test_creating_account_client.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from starknet_py.net.signer.stark_curve_signer import StarkCurveSigner
+
+
+@pytest.mark.asyncio
+async def test_creating_account_client(run_devnet):
+    # pylint: disable=import-outside-toplevel, unused-variable
+    # add to docs: start
+    from starknet_py.net import AccountClient, KeyPair
+    from starknet_py.net.models import StarknetChainId
+    from starknet_py.net.networks import TESTNET, MAINNET
+
+    # Creates an instance of account client which is already deployed
+    # mainnet
+    account_client_mainnet = AccountClient(
+        address="0x1234",
+        key_pair=KeyPair(private_key=123, public_key=456),
+        net=MAINNET,
+        chain=StarknetChainId.MAINNET,
+    )
+
+    # testnet
+    account_client_testnet = AccountClient(
+        address="0x1234",
+        key_pair=KeyPair(private_key=123, public_key=456),
+        net=TESTNET,
+        chain=StarknetChainId.TESTNET,
+    )
+
+    # There is another way of creating key_pair
+    key_pair = KeyPair.from_private_key(key=123)
+    # add to docs: end
+
+    signer = StarkCurveSigner("0x1234", key_pair, StarknetChainId.TESTNET)
+    # add to docs: start
+
+    # Instead of providing key_pair it is possible to specify a signer
+    account_client = AccountClient(
+        address="0x1234", signer=signer, net=TESTNET, chain=StarknetChainId.TESTNET
+    )
+    # add to docs: end
+    testnet = run_devnet
+    # add to docs: start
+
+    # Deploys an account on local network and returns an instance
+    account_client = await AccountClient.create_account(
+        net=testnet, chain=StarknetChainId.TESTNET
```

Comment:
same as above, chain is not required when you say net is TESTNET. And also - user doesn't know what `testnet` is here, there is no variable of that name in the docs from user's standpoint. Let's define it, and then switch it behind the scenes